extensions/mssql/src/credentialstore/credentialstore.ts

```diff
@@ -8,14 +8,14 @@ import { SqlOpsDataClient, ClientOptions, SqlOpsFeature } from 'dataprotocol-cli
 import * as path from 'path';
 import { IConfig, ServerProvider } from 'service-downloader';
 import { ServerOptions, RPCMessageType, ClientCapabilities, ServerCapabilities, TransportKind } from 'vscode-languageclient';
-import { Disposable } from 'vscode';
+import { Disposable, workspace } from 'vscode';
 import * as UUID from 'vscode-languageclient/lib/utils/uuid';
-
 import * as sqlops from 'sqlops';
 
 import * as Contracts from './contracts';
 import * as Constants from './constants';
 import * as Utils from '../utils';
+const findRemoveSync = require('find-remove');
```

Comment:
we generally prefer the `import` syntax above instead of `require`. Does this need to be `require` for any reason?